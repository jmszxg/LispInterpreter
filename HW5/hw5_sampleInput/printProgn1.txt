(print 
 (progn 
   "cheeseburger" "fries" "shake"
  )
)

