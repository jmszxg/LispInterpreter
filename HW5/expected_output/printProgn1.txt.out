"shake"

---- Completed parsing ----


Value of the expression is: "shake"

Bye!
